app/routes.py

```diff
@@ -1 +1,113 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
```

Comment:
Since this file only handles routes for tasks, we should rename the file to task_routes.py